--- mps_youtube/player.py.orig	2020-05-04 08:33:06.347820495 +0200
+++ mps_youtube/player.py	2020-05-04 08:36:58.187627290 +0200
@@ -275,7 +275,10 @@
     known_player = util.is_known_player(config.PLAYER.get)
     if known_player:
         pd = g.playerargs_defaults[known_player]
-        args.extend((pd["title"], song.title))
+	# Use new mpv syntax
+        # https://github.com/mps-youtube/mps-youtube/issues/1052
+        completetitle = '='.join((pd["title"], '"{0}"'.format(song.title)))
+        util.list_update(completetitle, args)
 
         if pd['geo'] not in args:
             geometry = config.WINDOW_SIZE.get or ""
@@ -287,7 +290,10 @@
                 geometry += xx + yy
 
             if geometry:
-                args.extend((pd['geo'], geometry))
+		# Use new mpv syntax
+                # See: https://github.com/mps-youtube/mps-youtube/issues/1052
+                newgeometry = '='.join((pd['geo'], geometry))
+                util.list_update(newgeometry, args)
 
         # handle no audio stream available
         if override == "a-v":
